e2e-tests/tfm/cypress/e2e/journeys/case/activities/activities.spec.js

```diff
@@ -28,7 +29,7 @@ context('Users can create and submit comments', () => {
       const otherActivity = {
         tfm: {
           activities: {
-            type: 'OTHER',
+            type: ACTIVITY_TYPES.OTHER,
```

Comment:
I can't tell if `"Other"` is a real activity type - it only seems to exist in tests (but the same is true of `"ACTIVITY"` and I've found that in the dev database!)